### PR TITLE
Chore - Schedule Sentry Tasks instantly

### PR DESCRIPTION
**Pull request by strseb**
## Description

I've been looking why we have so many "Abnormal" sessions in sentry and so little healthy ones. 
Problem is, session_end will attempt to send a request, if that fails we're abnormal. 

So let's try to do that immediatly instead of scheudling a new task. So we have better channces to get that out. 
This also makes the crash report screen pop instantly, which also is nice c:

### CODE REVIEW

File: src/apps/vpn/mozillavpn.cpp
Comment (vinocher) [inline src/apps/vpn/mozillavpn.cpp:901]: Isn't it possible to move this into SentryAdapter::init() for encapsulation? Also, a comment may be useful about why this is being done when Controller::readyToQuit is emitted.

File: src/shared/sentry/sentryadapter.h
Comment (vinocher) [inline src/shared/sentry/sentryadapter.h:64]: Do the comments above this line need be changed because this now called a bit earlier?

File: src/apps/vpn/mozillavpn.cpp
Comment (strseb) [reply src/apps/vpn/mozillavpn.cpp:901]: Not right now, however this can wait until this is merged:
https://github.com/mozilla-mobile/mozilla-vpn-client/pull/7520

Right now Sentry-Adapter sit's in the "shared" part, therefore it cannot know about the Controller.

File: src/shared/sentry/sentryadapter.cpp
Comment (lesleyjanenorton) [inline src/shared/sentry/sentryadapter.cpp:153]: https://github.com/mozilla-mobile/mozilla-vpn-client/blob/308692c7ee9d3bda41d8d6e4cd9af9dadc83c659/src/shared/taskscheduler.h#L20-L22
Footgun check: Is surety at 100%?

**Review by lesleyjanenorton — APPROVED** [review]

**Pull request merged**
